components/features/wallet/receive/index copy.tsx

```
@@ -0,0 +1,210 @@
+		setIsSharing(true);
+		try {
+			const uri = await captureRef(qrCodeRef, {
+				format: "png",
+				quality: 0.8,
+			});
+
+			const response = await fetch(uri);
+			const blob = await response.blob();
+			const file = new File([blob], "qr-code.png", { type: "image/png" });
+
+			if (navigator.canShare && navigator.canShare({ files: [file] })) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)